incident_response_checklist.md

```diff
@@ -14,6 +14,7 @@ For on-call engineers
         - Daytime: contact a support specialist for assistance (slack #dsde-comms or @dspsupportteam in #workbench-resilience). 
         - Off-hours: Contact the user and cc support@terra.bio . The email should explain the time the issue occurred, how the issue is impacting the user, and any relevant Terra details (workspace/submission/notebook/etc.) The Frontline support team will follow up with the user the next work day to answer any further questions.
     - Confirm that the "Users Informed" field has been updated on the Jira bug.
+- <input type='checkbox'> **If needed, open a Support Case with Google**  Follow steps in the [Google Support Case Creation](https://docs.google.com/document/d/1zSyuFJ7Qn7HfEVHssNF43VNyF-ox8I7ZaMBBXbbqt2g/edit#) document.
```

Comment:
Does this need to be a step in the checklist?  Isn't it technically a remediation step, and can it be linked to in the general playbook?
(trying to keep these checklists as light-weight as possible)

Reply:
Sure, makes sense with me.  The todo item specifically called out this checklist, which is why I made the change, but I'm all for keeping this streamlined.  I'll add it to the General Playbook

Reply:
Thank you! I believe there's a place for "what to do if it's a google issue"